spec/integration/default_spec.rb

```diff
@@ -51,9 +51,13 @@
   end
 end
 
+describe file('/var/itamae/backup/tmp') do
+  it { should be_directory }
+end
+
```

Comment:
I can't specify the backed-up file because file name is decided from date.
I try to use `commmand` resource with wildcard like below.

```ruby
describe command('cat /var/itamae/backup/tmp/file.itamae-*') do
  its(:stdout) { should match(/Hello World/) }
  its(:exit_status) { should eq(0) }
end
```